.gitpod.yml

```
@@ -0,0 +1,22 @@
+# List the start up tasks. You can start them in parallel in multiple terminals. See https://www.gitpod.io/docs/config-start-tasks/
+tasks:
```

```suggestion
tasks:
```https://github.com/babel/babel/tree/nicolo-ribaudo/setup-gitpod